client/src/js/services/receipts/modal/receiptModal.tmpl.html

```diff
@@ -14,15 +14,15 @@
       style="padding-top : 8px"
       class="pull-right">
     <span class="text-success">
-      <span class="glyphicon glyphicon-ok-circle"></span>
+      <span class="fa fa-check-circle-o"></span>
       {{ReceiptCtrl.createdKey | translate}}
     </span>
   </div>
 </div>
 
 <!-- Temporary Loading State -->
-<div style="padding : 10px;" ng-if="!ReceiptCtrl.receipt">
-  <p class="text-info"><span class="glyphicon glyphicon-hourglass"></span> {{"RECEIPTS.LOADING"}}</p>
+<div style="padding : 10px;" ng-if="!ReceiptCtrl.receipt" class="text-center">
+  <p class="text-info"><span class="fa fa-hourglass-end fa-spin"></span> {{ "RECEIPTS.LOADING" | translate }}</p>
```

Comment:
I've previously updated the bhLoadingButton component to use [a circle spinner](http://fontawesome.io/icon/circle-o-notch/). Just a note that at some point we should standardise how and when we show different types of progress.